### DISCUSSION TITLE
Is there any option where we can set the initial number of items to render(I know flatlist offers it)?

### DISCUSSION THREAD

**Discussion by bhushan-patil-official** [Shopify/flash-list#763] [Q&A]
I know the flatlist has the prop to mention the number of items to render but when I am using in flashlist its not rendering all the items. Its rendering the item based on the estimated size of the item. The thing is my list does not have a specific size of item. the Item consists of text, image, comments, so the size vaies depending upon the content is present on the item or not.
Even if I set a size say 200, when I scroll down the component which is scrolled above displays as the new component for split second and if I scroll down fast there is blank area.

Please help

**Comment by Yuniac** [comment]
Nothing still?

**Comment by RichardSleet** [comment]
Nothing still?
